dbn_upper_bound/python/mputility.py

```diff
@@ -9,20 +9,15 @@
 http://michaelnielsen.org/polymath1/index.php?title=Asymptotics_of_H_t''' 
 
 from mpmath import mp
-from dbn_upper_bound.python.constants import PI
 
 mp.dps=30
 
-PI = mp.mpf(PI)
```

Comment:
@km-git-acc 

Any reason why you chose to replace all instances of PI with mp.pi() instead of changing PI and PI_sq to

PI = mp.pi()
PI_sq = mp.pi() * mp.pi()